qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -202,6 +211,7 @@ def randomized_benchmarking_seq(nseeds: int = 1,
                                     Union[List[QuantumCircuit], List[Instruction],
                                           List[qiskit.quantum_info.operators.symplectic.Clifford],
                                           List[CNOTDihedral]]] = None,
+                                keep_original_interleaved_elem: Optional[bool] = False,
```

Comment:
Can this default to `True` to reproduce conventional behavior?

Reply:
The current behavior of RB is to convert the interleaved element to the group representation, which is what happens when  `keep_original_interleaved_elem == False`. I wish to keep it default in order to avoid impacting existing code.

Reply:
I'm now convinced that `True` is the correct default value, although it will require to pass `False` explicitly to the existing tests.